src/ORM/Table.php

```diff
@@ -524,13 +527,19 @@ public function connection(ConnectionInterface $connection = null)
      */
     public function getSchema()
     {
+        $logQueries = ConnectionManager::get('default')->logQueries();
```

Comment:
Many applications have more than just one connection. Couldn't this schema logging be done at a lower level? Perhaps in the SchemaCollection instead?